docs/sources/articles/authentication.md

```diff
@@ -1,31 +1,112 @@
-page_title: Running Docker with HTTPS
-page_description: How to setup and run Docker with HTTPS
-page_keywords: docker, docs, article, example, https, daemon, tls, ca, certificate
+page_title: Authenticating to the Docker daemon
+page_description: How to setup the Docker daemon with encryption and authentication
+page_keywords: docker, docs, article, example, https, daemon, tls, ca, certificate, authentication
 
 # Running Docker with https
 
 By default, Docker runs via a non-networked Unix socket. It can also
-optionally communicate using a HTTP socket.
+optionally communicate using a HTTP socket.  Enabling communication
+through HTTP will by default use TLS and require daemon configuration
+to allow client connections.
```

Comment:
I am thinking I should just remove this additional sentence for now since we pulled back on making it the default.  Not sure how to make this sentence useful if it is not describing the default behavior.